README.md

```diff
@@ -71,9 +75,11 @@ Below are the specific policies:
 | :---     | :---                    | :---   |
 | 2.x      | Dependencies `digest`, `pkcs8` and `rand_core` | Minor SemVer bump |
 
+TODO @tarcieri what about PEM?
```

Comment:
We only lists the crates / dependencies but not features as such here -

`pem` as a feature comes via `pkcs8` (unstable) crate via optional feature of `ed25519` which itself is stable on all-default-on -

Since it's not a crate or dependency itself unlike pkcs8 - theoretically we don't need to mention `pem` here, just `pkcs8` given it's optional dependency of `ed25519` thus exempted.

Given anyone who is using `pem` will inherently use `pkcs8` I don't see much problem not mentioning `pem` here ?